app/move/controllers/confirmation.js

```diff
@@ -0,0 +1,31 @@
+const { mountpath: movesUrl } = require('../../moves')
+const filters = require('../../../config/nunjucks/filters')
+
+function confirmation(req, res) {
+  const { move } = res.locals
+  const {
+    date,
+    person,
+    reference,
+    move_type: moveType,
+    to_location: toLocation,
+  } = move
+
+  const locals = {
+    moveReference: reference,
+    fullname: person.fullname,
+    location:
+      moveType === 'prison_recall'
+        ? req.t('fields::move_type.items.prison_recall.label')
+        : toLocation.title,
+    date: filters.formatDateWithDay(date),
+    action: {
+      createNewMoveUrl: '/move/new',
```

Comment:
I'd probably move this to the view too as that's what we've done elsewhere for now.

We might want to create some kind of URL/path helper in the future to ensure all our URLs are consistent but for now probably move this to the view.